jest.config.js

```diff
@@ -1,9 +1,6 @@
 module.exports = {
-  testMatch: ['**/+(*.)+(spec|test).+(ts|js)?(x)'],
-  transform: {
-    '^.+\\.(ts|js|html)$': 'ts-jest',
-  },
-  resolver: '@nrwl/jest/plugins/resolver',
-  moduleFileExtensions: ['ts', 'js', 'html'],
-  coverageReporters: ['html'],
+  projects: [
```

Comment:
The new `@nrwl/jest` package exports a `getJestProjects() function that can be used here.

```
projects: getJestProjects()
```

Reply:
done